### PR TITLE
Shield and Fireball VFX Upgrades

### PR DESCRIPTION
**Pull request by George540**
New Shield Shader for Magic Shield ability:
image

Bonus! New Light effect for Fireball ability:
image

### CODE REVIEW

File: Assets/Scripts/Combat/Combantants/Combatant.cs [hunk 1]
```diff
@@ -178,7 +178,7 @@ public void ApplyShield(int shieldHealth)
         if (MaxShieldPoints == 0)
         {
             Shield.SetActive(true);
-            Shield.transform.position = transform.position;
+            Shield.transform.position = transform.position + new Vector3(0.0f, 2.0f, 0.0f);
```
Comment (armandorusso) [inline Assets/Scripts/Combat/Combantants/Combatant.cs:181]: Make the ``new Vector3(0.0f, 2.0f, 0.0f);`` a variable and call it _Offset_. That way we can easily change it whenever we want.

**Review by armandorusso — CHANGES_REQUESTED** [review]
Looks amazing! Love the shield VFX and fireball lighting changes! Just a small code fix.
Also: would it be possible to make the fireball lighting a tiny bit more intense? Not by much though.

File: Assets/Scripts/Combat/Combantants/Combatant.cs [hunk 2]
```diff
@@ -178,7 +179,7 @@ public void ApplyShield(int shieldHealth)
         if (MaxShieldPoints == 0)
         {
             Shield.SetActive(true);
-            Shield.transform.position = transform.position;
+            Shield.transform.position = transform.position + new Vector3(0.0f, ShieldPositionVerticalOffset, 0.0f);
```
Comment (armandorusso) [inline Assets/Scripts/Combat/Combantants/Combatant.cs:182]: What I meant by my previous comment in the other commit was make the whole line ``new Vector3(0.0f, ShieldPositionVerticalOffset, 0.0f);`` a variable, not just the y position.

**Review by armandorusso — APPROVED** [review]
Looks better! I'll approve it cause its just a minor fix really.

File: Assets/Scripts/Combat/Combantants/Combatant.cs (hunk 2 above)
Comment (George540) [reply Assets/Scripts/Combat/Combantants/Combatant.cs:182]: No, a vector3 cannot be stored as a constant

**Review by eli-lam — APPROVED** [review]

**Pull request merged**